src/pcapi/local_providers/provider_api/provider_api_stocks.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy.orm.util import aliased
+
+from pcapi.models import VenueProvider
+from pcapi.models.provider import Provider
+from pcapi.utils.logger import logger
+
+from . import synchronize_provider_api
+
+
+def synchronize_stocks() -> None:
+    # Alias for provider table, and explicit "on" clause for the "join", are mandatory because
+    # VenueProvider model already has a "select" on the provider table for its polymorphic query
+    provider_alias = aliased(Provider)
+
+    venue_providers = (
+        VenueProvider.query.join(provider_alias, provider_alias.id == VenueProvider.providerId)
+        .filter(provider_alias.apiUrl != None)
+        .filter(VenueProvider.isActive == True)
```

Comment:
micro-détail : 

```suggestion
        .filter(provider_alias.apiUrl.isnot(None))
        .filter(VenueProvider.isActive.is_(True))
```

Les 2 syntaxes sont bonnes, mais celle avec `==` et `!=` génère des alertes pylint (qu'on ignore actuellement, mais que j'aimerais essayer de réactiver).